tests/navigationBar/NavBar.test.ts

```diff
@@ -0,0 +1,125 @@
+import { test, expect } from '@playwright/test';
+
+/**
+ * Test About button stays on the landing page
+ */
+test('Test About button stays on the landing page', async ({ page }) => {
+  await page.goto('https://techstartucalgary.com/');
+  await page.getByText('About').click();
+  await expect(page).toHaveURL('https://techstartucalgary.com/');
+  await page.close();
+
+});
+
+/**
+ * Test Team button navigates to the team page and shows Founder
+ */
+test('Test Team button navigates to the team page and shows Founder', async ({ page }) => {
```

Comment:
Could you attach a screenshot showing all the tests passing? A few of these tests are not passing on my end and I would like to crosscheck which ones before investigating more

Reply:
After making the change from your other comment here is my result. You're right some of the tests fails.
<img width="608" alt="image" src="https://github.com/user-attachments/assets/a7852854-17aa-4566-aa52-5a30452721c9">
<img width="620" alt="image" src="https://github.com/user-attachments/assets/67a1ca21-5d9d-4a34-bdba-6ad13509a901">
<img width="626" alt="image" src="https://github.com/user-attachments/assets/39c3b40c-a6c7-4c81-8459-09286d31a638">

Reply:
Im trying to make it pass. Im not sure why its failing now

Reply:
I just remembered, you have to run the tests on localhost. I added a much of `data-testid` which don't exist on techstart.com

Reply:
I've pushed my branch - this is the latest error Im getting
<img width="820" alt="image" src="https://github.com/user-attachments/assets/730cac0e-68a7-4b62-94af-706242ad4dc2">

Reply:
Fixed the (hopefully. Used a promise seems to work
<img width="959" alt="image" src="https://github.com/user-attachments/assets/a58196af-f64f-405c-9ff2-cd5af0206ceb">